permission_handler_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,20 @@ description: A common platform interface for the permission_handler plugin.
 homepage: https://github.com/baseflow/flutter-permission-handler/tree/master/permission_handler_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.2
+version: 3.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
   meta: ^1.0.5
-  plugin_platform_interface: ^1.0.2
+  plugin_platform_interface: ^1.1.0-nullsafety
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mockito: ^4.1.1
-  effective_dart: ^1.2.1
+  mockito: ^5.0.0-nullsafety
+  effective_dart: ^1.3.0
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Let's set this to: ">=2.12.0-259.9.beta <3.0.0". 

The Dart team has marked this version as the stable NNBD version.